test.js

```diff
@@ -69,42 +68,66 @@ async function gruntDiff() {
     'node',
     './node_modules/grunt/bin/grunt',
     'diff',
-    shouldUseOutputDir && `--outputdir=${outputDir}`
+    Boolean(process.env.npm_config_outputdir) && `--outputdir=${argumentValues.outputdir}`
   ].filter(Boolean));
 };
 
 async function gruntServer() {
-  return backgroundSpawn('node', './node_modules/grunt/bin/grunt', 'server-silent', 'run', `--outputdir=${outputDir}`);
+  return backgroundSpawn('node', './node_modules/grunt/bin/grunt', 'server-silent', 'run', `--outputdir=${argumentValues.outputdir}`);
 };
 
 async function waitForGruntServer() {
   return waitForExec('node', './node_modules/wait-on/bin/wait-on', 'http://localhost:9001');
 };
 
 async function cypressRun() {
-  return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run');
+  return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run', '--spec', `${argumentValues.testfiles}`);
 };
 
 async function jestRun() {
-  config.testEnvironmentOptions.outputDir = outputDir;
+  config.testEnvironmentOptions.outputDir = argumentValues.outputdir;
+
+  // Limit the tests if a certain set are passed in
+  if (argumentValues.testfiles) {
+    config.testMatch = [argumentValues.testfiles.split(',')];
+  }
+
   return jest.runCLI(config, [process.cwd().replace(/\\/g, '/')]);
 };
 
 async function jestClear() {
   return asyncSpawn('node', './node_modules/jest/bin/jest', '--clearCache');
 };
 
+const acceptedArgs = [
+  'outputdir',
+  'skipinstall',
+  'testfiles'
+];
+
+const argumentValues = {
+  outputdir: (process.env.npm_config_outputdir || './build/'),
+  skipinstall: false,
+  testfiles: './test/**'
```

Comment:
default jest tests fail as the default glob should be more specific `'**/test/unit/**/*.js?(x)'`, fails because of line 91

```suggestion
  testfiles: null
```